tests/filtering_test.cc

```diff
@@ -791,6 +791,13 @@ SEASTAR_TEST_CASE(test_allow_filtering_with_secondary_index) {
                 }
             });
         });
+
+        eventually([&] {
+            auto msg = e.execute_cql("SELECT SUM(e) FROM t WHERE c = 5 AND b = 911 ALLOW FILTERING;").get0();
+            assert_that(msg).is_rows().with_rows({{ int32_type->decompose(0), {} }});
```

Comment:
Hmm, the sum here is 0? Does anything match in this filter?

Reply:
Precisely, no rows have `b = 911`, so the sum is 0.